devfile.yaml

```diff
@@ -0,0 +1,88 @@
+schemaVersion: 2.1.0
+metadata:
+  name: scala-sbt
+attributes:
+  che-theia.eclipse.org/sidecar-policy: USE_DEV_CONTAINER
+components:
+  - name: tools
+    container:
+      image: quay.io/eclipse/che-java11-maven:next
+      env:
+        - name: JDK_JAVA_OPTIONS
+          value: >-
+            -XX:MaxRAMPercentage=80 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom
+      endpoints:
+        - exposure: none
+          name: debug
+          protocol: tcp
+          targetPort: 5005
+        - exposure: public
+          name: 8080-tcp
+          protocol: http
+          targetPort: 8080
+      volumeMounts:
+        - name: sbt
+          path: /home/user/.sbt
+        - name: ivy2
+          path: /home/user/.ivy2
+        - name: coursier
+          path: /home/user/.cache/coursier
+        - name: metals
+          path: /home/user/.cache/metals
+        - name: bloop
+          path: /home/user/.cache/bloop
+      memoryLimit: 5Gi
+      mountSources: true
+  - name: sbt
+    volume:
+      size: 1G
+  - name: ivy2
+    volume:
+      size: 1G
+  - name: coursier
+    volume:
+      size: 1G
+  - name: metals
+    volume:
+      size: 1G
+  - name: bloop
+    volume:
+      size: 1G
```

Comment:
It's not possible to start the workspace because of 
```
Error provisioning storage: Could not rewrite container volume mounts: container 'tools' references undefined volume 'cache'

Initializing DevWorkspace

Error provisioning storage: Could not rewrite container volume mounts: container 'tools' references undefined volume 'cache'
```
Try to add volume
```
- name: cache
  path: /home/user/.cache
```
and
```
  - name: ivy2
    volume:
      size: 1G
```

Reply:
Done. Thanks for the tip. Where can I see if it works or not?

Reply:
To deploy Che with Devworkspace engine you need to select a dedicated OperatorHub channel. Devsandbox doesn't provide it yet. Some more info for Che: https://che.eclipseprojects.io/2021/10/12/@mario.loriedo-devfile-v2-and-the-devworkspace-operator-p1.html